sdk/go/auto/optrefresh/optrefresh.go

```diff
@@ -128,6 +132,47 @@ type Options struct {
 	ShowSecrets *bool
 }
 
+func (o *Options) ToCmd(isPreview bool, remoteArgs []string, program pulumi.RunFunc) []string {
```

Comment:
Kind of a bummer we have to publicly export this internal implementation detail. Should we move it into the package where it's used (i.e. in `sdk/go/auto/stack.go`) so we can keep it private?